src/request/RequestHandler.ts

```diff
@@ -10,6 +11,22 @@ import { RequestContext } from "./RequestContext";
 import { Response as CosmosResponse } from "./Response";
 import { TimeoutError } from "./TimeoutError";
 
+let defaultHttpAgent: Agent;
+let defaultHttpsAgent: Agent;
+
+if (isNode) {
```

Comment:
notion: it might be nice to just rip this into its own file since it doesn't have much to do with RequestHandler